dz_3/dz_3_2.py

```diff
@@ -0,0 +1,33 @@
+"""
+Задача 18: 
+Требуется найти в массиве A самый близкий по величине элемент к заданному числу X. 
+Если таких элементов несколько, вы вывести один любой. 
+Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию). 
+Последняя строка содержит число X
+Пример:
+5
+    7 -2 3 5 1
+    6
+    -> 7
+"""
+
+import random
+
+n = int(input("Введите количество элементов массива: "))
+x = int(input("Введите искомое чило: "))
+
+arr = [random.randint(-10, 10) for i in range(n)]
+print(arr)
+
+
+for j in range(0,100):
```

Comment:
Она что направлен этот цикл? Да и ноль передавать не обязательно, если вы начинаете с нуля.
```suggestion
for j in range(100):
```